lib/attribute_normalizer/extras.rb

```diff
@@ -4,12 +4,16 @@
 require "attribute_normalizer/normalizers/spaceless_normalizer"
 require "attribute_normalizer/normalizers/postal_code_normalizer"
 require "attribute_normalizer/normalizers/float_normalizer"
+require "attribute_normalizer/normalizers/downcase_normalizer"
+require "attribute_normalizer/normalizers/upcase_normalizer"
 
 extras = {
   gsub:  AttributeNormalizer::Normalizers::GsubNormalizer,
   postal_code: AttributeNormalizer::Normalizers::PostalCodeNormalizer,
   spaceless: AttributeNormalizer::Normalizers::SpacelessNormalizer,
-  float: AttributeNormalizer::Normalizers::FloatNormalizer
+  float: AttributeNormalizer::Normalizers::FloatNormalizer,
+  downcase: AttributeNormalizer::Normalizers::DowncaseNormalizer,
+  uppercase: AttributeNormalizer::Normalizers::UpcaseNormalizer
```

Comment:
What do you think of calling this simply: `upcase`? Then it would match the class name.

Reply:
I missed this one when I was changing it :). Done!